cmd/checks/time/time_darwin.go

```diff
@@ -18,6 +18,6 @@ package time
 import "github.com/spf13/cobra"
 
 // Add doesn't add this check on darwin b/c it's not supported
```

Comment:
```suggestion
// Register doesn't add this check on darwin b/c it's not supported
```

Reply:
Looks like I missed 50% of the necessary renames 🤦‍♂️ Nice catch!

Reply:
Looks like I missed 50% of the necessary renames 🤦‍♂️ Nice catch!